frontend/src/app/shared/slide-view/slide-view.component.html

```diff
@@ -7,14 +7,21 @@
   <nav class="nav-tabs">
     <a
       *ngFor="let tab of tabs"
-      class="tab"
+      class="tab for-panel"
       [class.active]="tab.active"
       [title]="tab.title"
       (click)="selectTab(tab.id)"
       tabindex="0"
     >
       <app-icon [iconType]="tab.icon"></app-icon>
     </a>
+    <a *ngIf="(compoundComparisonCount$ | async) > 0"
```

Comment:
Should this be a button (or role="button") for accesability maybe?

Reply:
There 's a case to be made for this, and I don't disagree.
But then I think something similar should be done as well for the other navigation "tags"/"buttons". 

To get even more specific, if we were strict with regard to the 'aria' semantics/"role" attribute usage here, I imagine it can be argued further that we should really use them to distinguish between this new button and the navigational tabs, e.g. by using role="tab" for the others, and role="button" for this one.

For now, let's go with setting `role="button"` on the new element?
This, along with some other UI discrepancies, I believe are going to be better addressed by some more concerted effort in the near future, seeing as it's been discussed during refinement, lately.